pytheus/backends.py

```diff
@@ -126,16 +135,45 @@ def get(self) -> float:
 
 
 class MultipleProcessRedisBackend(Backend):
-    """Provides a multi-process backend that uses Redis."""
+    """
+    Provides a multi-process backend that uses Redis.
+    Single dimension metrics will be stored as list (ex. Counter) while labelled
+    metrics will be stored in an hash.
+    Currently is very naive, and there is a lot of room for improvement.
+    (ex. maybe store all dimensions in the same hash and be smart on retrieving everything...)
+    Note: currently not expiring keys
+    """
+
+    CONNECTION_POOL: redis.Redis = None
+
+    def __init__(self, config: BackendConfig, metric: 'Metric') -> None:
+        super().__init__(config, metric)
+        self._key_name = self.metric._collector.name
+        self._labels_hash = None
+        if self.metric._labels:
+            self._labels_hash = '-'.join(sorted(self.metric._labels.values()))
+
+        if self.CONNECTION_POOL is None:
+            MultipleProcessRedisBackend.CONNECTION_POOL = redis.Redis(
+                **config,
+                decode_responses=True,
+            )
 
-    def is_valid_config(self, config: BackendConfig) -> True:
+    def is_valid_config(self, config: BackendConfig) -> bool:
         return True  # TODO
 
     def inc(self, value: float) -> None:
-        pass  # TODO
+        if self._labels_hash:
```

Comment:
currently using either a list or an hash, depending if it's a dimensioned metric